tests/playwright/pages/editor-page.js

```diff
@@ -25,14 +26,15 @@ module.exports = class EditorPage extends BasePage {
 			return url.replace( regex, `${ date.getFullYear() }/${ month }` );
 		};
 		templateData.content[ 0 ].elements.forEach( ( el ) => {
-			if ( 'image' in el.settings ) {
-				el.settings.image.url = replaceUrl( el.settings.image.url );
-			}
 			if ( 'carousel' in el.settings ) {
 				for ( let i = 0; i < el.settings.carousel.length; i++ ) {
 					el.settings.carousel[ i ].url = replaceUrl( el.settings.carousel[ i ].url );
 				}
 			}
+			const key = _.findKey( el.settings, 'url' );
```

Comment:
Can we use something like this instead of adding the Lodash library?
Taken from: https://stackoverflow.com/questions/70673437/lodash-findkey-equivalent-in-javascript-or-jquery
```javascript
const findKey = (obj, predicate = o => o) => Object.keys(obj)
    .find(key => predicate(obj[key], key, obj))
```

Reply:
we can, but I see no value in re-inventing loadash. `_.findKey( el.settings, 'url' );` looks more cleaner. Also, lodash has a lot of useful functions for javascript. It's hard to find a project, that doesn't use it. In some test frameworks like Cypress it is available out of the box. Also, playwright use it a lot.